core/test/functional/api/v0.1/public_api_spec.js

```diff
@@ -270,6 +270,45 @@ describe('Public API', function () {
             });
     });
 
+    it('browse posts: request only url fields', function (done) {
+        request.get(localUtils.API.getApiQuery('posts/?client_id=ghost-admin&client_secret=not_available&fields=url'))
+            .set('Origin', testUtils.API.getURL())
+            .expect('Content-Type', /json/)
+            .expect('Cache-Control', testUtils.cacheRules.private)
+            .expect(200)
+            .end(function (err, res) {
+                if (err) {
+                    return done(err);
+                }
+
+                should.exist(res.body.posts);
+
+                should.equal(res.body.posts[0].id, undefined);
```

Comment:
If you would use `checkResponse`, you can tell this utility helper to only receive the url field using `onlyProperties`.

Reply:
Nice tip :) Will change this asap